Extensions/PrimitiveDrawing/Extension.cpp

```diff
@@ -168,9 +187,194 @@ void DeclarePrimitiveDrawingExtension(gd::PlatformExtension& extension) {
+                _("Add to a path a line to the position _PARAM1_;_PARAM2_ "
+                  " with _PARAM0_"),
+                _("Advanced"),
+                "res/actions/line24.png",
+                "res/actions/line.png")
+
+      .AddParameter("object", _("Shape Painter object"), "Drawer")
+      .AddParameter("expression", _("X position of start point"))
+      .AddParameter("expression", _("Y position of start point"))
+      .SetFunctionName("drawPathLineTo")
+      .SetIncludeFile("PrimitiveDrawing/ShapePainterObject.h");
+
+  obj.AddAction("PathBezierCurveTo",
+                _("Path bezier curve to"),
+                _("Add to a path a bezier curve to a position, the origin come from the previous action or from BeginFillPath or PathMoveTo, by default if nothing exist, start position will be the object's position."),
+                _("Add to a path a bezier curve to the position _PARAM5_;_PARAM6_ (first control point: _PARAM1_;_PARAM2_, second control point: _PARAM3_;_PARAM4_) "
+                  " with _PARAM0_"),
+                _("Advanced"),
+                "res/actions/delete24.png",
+                "res/actions/delete.png")
+
+      .AddParameter("object", _("Shape Painter object"), "Drawer")
+      .AddParameter("expression", _("First control point x"))
+      .AddParameter("expression", _("First control point y"))
+      .AddParameter("expression", _("Second Control point x"))
+      .AddParameter("expression", _("Second Control point y"))
+      .AddParameter("expression", _("Destination point x"))
+      .AddParameter("expression", _("Destination point y"))
+      .SetFunctionName("drawPathBezierCurveTo")
+      .SetIncludeFile("PrimitiveDrawing/ShapePainterObject.h");
+/*
+    obj.AddAction("drawPathArcTo",
+                _("drawPathArcTo"),
+                _("drawPathArcTo"),
+                _("drawPathArcTo"
+                " with _PARAM0_"),
+                _("Advanced"),
+                "res/actions/arc24.png",
+                "res/actions/arc.png")
+
+      .AddParameter("object", _("Shape Painter object"), "Drawer")
+      .AddParameter("expression", _("X control point"))
+      .AddParameter("expression", _("Y control point"))
+      .AddParameter("expression", _("X coordinate end arc"))
+      .AddParameter("expression", _("Y coordinate end arc"))
+      .AddParameter("expression", _("Radius (in pixels)"))
+      .SetFunctionName("drawPathArcTo")
+      .SetIncludeFile("PrimitiveDrawing/ShapePainterObject.h");
+*/
+
+  obj.AddAction("PathArc",
+                _("Path arc to"),
+                _("Add to a path an arc to a position, the origin come from the previous action or from BeginFillPath or PathMoveTo, by default if nothing exist, start position will be the object's position."),
+                _("Add to a path an arc at the position _PARAM1_;_PARAM2_ (radius: _PARAM3_,start angle: _PARAM4_, end angles: _PARAM5_,anticlockwise: _PARAM6_ "
```

Comment:
space before "start angle"
space before anticlockwise
no s to "end angles"
no space at the end and close the parenthesis